src/Kafka/ConsumerGroups/components/ConsumerGroupResetOffset.tsx

```diff
@@ -0,0 +1,262 @@
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import { TopicSelect } from "./TopicSelect";
+import { OffsetSelect } from "./OffsetSelect";
+
+export type ConsumerRow = Consumer & {
+  selected?: boolean;
+};
+
+export type ConsumerGroupResetOffsetProps = {
+  isModalOpen: boolean;
+  isDisconnected: boolean;
+  groupId: string;
+  topics: string[];
+  selectedTopic: string;
+  customOffsetValue: string;
+  setcustomOffsetValue: (customOffsetValue: string) => void;
+  consumers: ConsumerRow[];
+  isSelected: boolean;
+  onClickClose: () => void;
+  onClickResetOffset: () => void;
+  onChangeTopic: (value: string) => void;
+  selectedOffset: OffsetValue;
+  onChangeOffsetValue: (value: OffsetValue) => void;
+  confirmCheckboxChecked: boolean;
+  onConfirmationChange: (value: boolean) => void;
+  SelectAllConsumer: (consumer: ConsumerRow[]) => void;
+  onSelectAll: (isSelected: boolean) => void;
+  onSelectRow: (index: number, isSelected: boolean) => void;
+};
+
+export const ConsumerGroupResetOffset: FunctionComponent<
+  ConsumerGroupResetOffsetProps
+> = ({
+  isDisconnected,
+  groupId,
+  topics,
+  customOffsetValue,
+  setcustomOffsetValue,
+  consumers,
+  onClickClose,
+  isModalOpen,
+  onClickResetOffset,
+  selectedTopic,
+  onChangeTopic,
+  selectedOffset,
+  onChangeOffsetValue,
+  confirmCheckboxChecked,
+  onConfirmationChange,
+  isSelected,
+  onSelectAll,
+  onSelectRow,
+}) => {
+  const { t } = useTranslation(["kafka"]);
+
+  const isResetOffsetDisabled = (): boolean => {
+    return (
+      !selectedTopic ||
+      !selectedOffset ||
+      !confirmCheckboxChecked ||
+      !isDisconnected ||
+      consumers.filter(({ selected }) => selected === true).length === 0
+    );
+  };
+
+  const tableColumns = {
+    partition: t("consumerGroup.partition"),
+    clientId: `${t("consumerGroup.client_id")} + ${t(
+      "consumerGroup.member_id"
+    )}`,
+    current_offset: t("consumerGroup.current_offset"),
+    log_end_offset: t("consumerGroup.log_end_offset"),
+    offset_lag: t("consumerGroup.offset_lag"),
+    new_offset: t("consumerGroup.new_offset"),
+  };
+
+  return (
+    <Modal
+      variant={ModalVariant.large}
+      isOpen={isModalOpen}
+      aria-label={t("consumerGroup.reset_offset_modal_label")}
+      title={t("consumerGroup.reset_offset")}
+      showClose={true}
+      aria-describedby="modal-message"
+      actions={[
+        <Button
+          variant="danger"
+          key={1}
+          isDisabled={isResetOffsetDisabled()}
+          onClick={onClickResetOffset}
+        >
+          {t("consumerGroup.reset_offset")}
+        </Button>,
+        <Button variant="link" key={2} onClick={onClickClose}>
+          {t("common:cancel")}
+        </Button>,
+      ]}
+    >
+      <Stack hasGutter>
+        <StackItem>
+          <Form isHorizontal>
+            <FormGroup label="Consumer group" fieldId="horizontal-form-name">
```

Comment:
Using a `<Title>` component here doesn't seem appropriate.

One option is to just include `{groupID}` as the direct child of `<FormGroup hasNoPaddingTop>`. The `hasNoPaddingTop` removes the top padding so that the label lines up better with this text.

Another option is to use the read-only styles we have in the Kafka topic page. We have a custom class for `<FormGroup>` - `.kafka-ui-form-group--readonly` - which lets us use a read-only input to display read-only values in a similar visual style. 

![image](https://user-images.githubusercontent.com/21063328/171760173-4dd6a957-965b-4a18-a61c-fd414e0b138b.png)

P.S. I would definitely recommend the second pattern if we have a really long form where some properties are read-only and some are editable (like the edit page). But in this case, the user pre-selected the consumer group, so it's just here to provide additional context.

Reply:
@jenn as per the @riccardo-forina comment, I removed the width in the select component, and also replaced the title component with Textinput, here is the updated story https://619e1921b3e38b003afeb2be-ypuxtqyvbq.chromatic.com/?path=/story/kafka-consumergroups-components-consumergroupresetoffset--reset-offset-when-offset-is-absolute